src/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -53,25 +53,33 @@ public object CreateFake(Type typeOfFake, Action<IFakeOptions> optionsBuilder)
         /// </returns>
         public IList<T> CollectionOfFake<T>(int numberOfFakes)
         {
+            return this.CollectionOfFake<T>(numberOfFakes, x => { });
+        }
+
+        public IList<T> CollectionOfFake<T>(int numberOfFakes, Action<IFakeOptions<T>> optionsBuilder)
+        {
             var result = new List<T>();
 
             for (var i = 0; i < numberOfFakes; i++)
             {
-                result.Add(this.CreateFake<T>(x => { }));
+                result.Add(this.CreateFake(optionsBuilder));
             }
 
             return result;
         }
 
-        public IList<object> CollectionOfFake(Type typeOfFake, int numberOfFakes)
+        public IList<object> CollectionOfFake(Type typeOfFakes, int numberOfFakes)

```

Comment:
The parameter in `A` is named `typeOfFake` so I think we should be consistent.

If we'd rather pluralise it in `A` then I think that's OK, but only because we haven't released those overloads yet, as this could be called a breaking change (I'm not sure what our breaking/non-breaking policy is on parameter name changes).

Whatever the decision, I'd prefer this commit to be consistent with `A` and remain singular. This would avoid having this unrelated change here. If we decide to pluralise both here and in `A`, that should be a separate commit.